.github/workflows/yum-release.yml

```diff
@@ -0,0 +1,66 @@
+name: Create Yum Release
+
+on:
+  workflow_dispatch:
+  release:
+    types:
+      - created
+
+jobs:
+  release:
+    runs-on: ubuntu-latest
+    steps:
+      -
+        name: Checkout
+        uses: actions/checkout@v2
+        with:
+          fetch-depth: 0
+      -
+        name: change release ver
+        run: |
+          # Set release version based on github ref
+          # RELEASE_VERSION=v0.1.2
+          RELEASE_VERSION=$(echo ${GITHUB_REF} | rev | cut -d'/' -f 1 | rev )
```

Comment:
Why are we doing it this way? Just curious...

Reply:
We need that format of version (v0.1.2) for the package manager, and the most surefire way of getting it is from the github ref supplied by a release action. [GH Release Action Docs](https://docs.github.com/en/actions/reference/events-that-trigger-workflows#release)